tests/unit/lms/services/ltia_http_test.py

```diff
@@ -18,7 +18,6 @@ def test_request(self, svc, jwt_service, application_instance, uuid, http_servic
                 "aud": application_instance.lti_registration.token_url,
                 "exp": datetime(2022, 4, 4, 1, 0),
                 "iat": datetime(2022, 4, 4, 0, 0),
-                "nonce": uuid.uuid4.return_value.hex,
```

Comment:
Nonce was not part of of the `client_credentials` parameters but we were sending it because it was in the defaults.